src/app/core/pipes/pipes.module.ts

```diff
@@ -0,0 +1,17 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+
+// import { PersonalAchievementsDatePipe } from './personal-achievements-date.pipe';
+
+@NgModule({
+  declarations: [
+    // PersonalAchievementsDatePipe
+  ],
+  imports: [
+    CommonModule
+  ],
+  exports: [
+    // PersonalAchievementsDatePipe
```

Comment:
remove commented code

Reply:
I removed this module

Reply:
> I removed this module

But you still have this module imported in other modules. If you want to remove this file - please fully remove it, because i do not see that you removed it

Reply:
Yes, I did it